src/assets/img_wip.svg

```diff
@@ -0,0 +1,64 @@
+<svg width="360" height="264" viewBox="0 0 360 264" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
(nit) please rename this file to: `workInProgress.svg`

Reply:
**One more thing**: this file should probably be refactored to come from the `b2c-mapp-ui-assets` repo, but only after you've finished changing your Vue markup to directly use this component, and figured out the correct CSS for resizing it (because that may require some small changes in this SVG markup).